README.md

```diff
@@ -144,6 +146,15 @@ Where `no-foo` is the rule name, `Disallow use of foo` is the rule description,
 | `name` | `# no-foo` |
 `prefix-name` | `# test/no-foo` |
 
+### Configuration file
+
+There are a few ways to create a config file:
+
+- An object exported by `.eslint-doc-generatorrc.js`, `.eslint-doc-generatorrc.json`, or any other config file format/name supported by [cosmiconfig](https://github.com/davidtheclark/cosmiconfig#searchplaces)
```

Comment:
Do you think we should an example of a config file?

Reply:
Good idea, added.